texastribune/salesforce-stripe
# Upgrade to CircleCI 2

#### What's this PR do?
What the title says.

#### Why are we doing this? How does it help us?
CircleCI v1 is losing support soon.

#### How should this be manually tested?
1. Install the CircleCI command line client by following the instructions here: https://circleci.com/docs/2.0/local-jobs/
2. `circleci config validate -c .circleci/config.yml`
3. `circleci build` - this step should take a while and you'll see some expected errors like this: `Error: FAILED with error not supported`. But as long as you see `Success!` at the end then it worked.

#### Have automated tests been added?
Tests for the tests? No.

#### Has this been tested on mobile?
NA.

#### Are there performance implications?
No.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/1181274383

#### How should this change be communicated to end users?
NA.

#### Next steps?
Find out what other apps need to be upgraded.

#### Smells?
NA.

#### TODOs:
NA.

#### Has the relevant documentation/wiki been updated?
NA.

#### Technical debt note
Less.
